packages/battery_plus/pubspec.yaml

```diff
@@ -14,31 +14,30 @@ flutter:
       ios:
         pluginClass: FLTBatteryPlusPlugin
       linux:
-        default_package: battery_plus_linux
+        dartPluginClass: BatteryPlusLinux
       macos:
-        default_package: battery_plus_macos
+        pluginClass: BatteryPlusMacosPlugin
       web:
-        default_package: battery_plus_web
+        pluginClass: BatteryPlusPlugin
+        fileName: src/battery_plus_web.dart
```

Comment:
Can the web plugin be renamed to `BatteryPlusWebPlugin` (and do we need `fileName`)?

Reply:
Renamed! But the fileName seems to be necessary

Reply:
Sorry, I updated my comment. It's no plugin, right? Could it just be `BatteryPlusWeb`?

Reply:
I would either add it to the linux one, or remove it from web. Would be happy with either option.
Both would make sense (as they're pure Dart and therefore technically no plugins (if I'm correct), it could be omitted - but since all platform implementations are called plugin, it would also make sense to name all consistently).

Reply:
I renamed the Linux one for consistency with the other 5 platforms

Reply:
Thanks a lot 👍🏽